core/src/main/java/org/apache/calcite/rel/rules/PruneEmptyRules.java

```diff
@@ -479,13 +481,24 @@ public interface JoinLeftEmptyRuleConfig extends PruneEmptyRule.Config {
     @Override default PruneEmptyRule toRule() {
       return new PruneEmptyRule(this) {
         @Override public void onMatch(RelOptRuleCall call) {
-          Join join = call.rel(0);
+          final Join join = call.rel(0);
+          final Values empty = call.rel(1);
+          final RelNode right = call.rel(2);
+          final RelBuilder b = call.builder();
```

Comment:
Nitpick: in the codebase this is generally called `relBuilder`, the method is not big but you might wonder what `b` is in the rest of the code, while with `relBuilder` it will be pretty clear

Reply:
That's reasonable. I'll apply the change to my branch, and will merge tomorrow when my batch of changes passes.

Reply:
Renamed to `relBuilder`